### ISSUE TITLE
Source for Fuschia Rust Ban

### ISSUE THREAD

**Issue by carbotaniuman** [slightknack/rust-abi-wiki#3]
> Recently, the Fuschia OS Team at Google decided to ban Rust's for use in Fuschia microkernel

This is weird because it used in Fuschia

**Comment by slightknack** [comment]
You're right, this statement is incorrect. `InitialProposal.md` Is the initial proposal, with minor edits, as it was originally published. Fuchsia not banning Rust was first addressed [here](https://internals.rust-lang.org/t/a-stable-modular-abi-for-rust/12347/45), and then later acknowledged [here](https://internals.rust-lang.org/t/a-stable-modular-abi-for-rust/12347/58). I think an addendum should be added clarifying that that statement is incorrect or the removal of that phrase would be a good idea. Would you mind opening a quick PR pointing out that this is indeed the case? Just a small blockquote pointing to the discussion with a comment will do. If you're busy, I wouldn't mind opening a quick PR. Thanks for bringing that to my attention 😄

**Comment by slightknack** [comment]
By the way, here's Fuchsia's current [Programming Language Policy](https://fuchsia.googlesource.com/fuchsia/+/refs/heads/master/docs/contribute/governance/policy/programming_languages.md). From the policy:

> - Rust is approved for use throughout the Fuchsia Platform Source Tree, with the following exceptions:
>     - *kernel*. The Zircon kernel is built using a restricted set of technologies that have established industry track records of being used in production operating systems.

Rust is not approved for use withing the Zircon microkernel, though this seems to be more because of it's novelty. A stable ABI is a good step towards maturing a language, which is what I believe the original reasoning behind this statement to be. Hope that this clears things up.
